internal/photoprism/convert_sidecar_json.go

```diff
@@ -58,7 +58,39 @@ func (c *Convert) ToJson(f *MediaFile, force bool) (jsonName string, err error)
 
 	// Check if file exists.
 	if !fs.FileExists(jsonName) {
-		return "", fmt.Errorf("exiftool: failed creating %s", filepath.Base(jsonName))
+		return "", fmt.Errorf(
+			"exiftool: failed creating %s", filepath.Base(jsonName),
+		)
+	}
+
+	// attempt to extract the video file if it exists
+	var extractCmd *exec.Cmd
+	path := filepath.Join(
+		Config().SidecarPath(), f.RootRelPath(), "%f"+fs.ExtMP4,
```

Comment:
Is there a way to tell if the embedded file is actually an MP4 video? This might change in the future when new formats are introduced.